src/main/java/com/checkmarx/flow/CxFlowRunner.java

```diff
@@ -262,15 +264,24 @@ private void commandLineRunner(ApplicationArguments args) throws ExitThrowable {
                 repoType = ScanRequest.Repository.GITHUB;
 
                 if(ScanUtils.empty(namespace) ||ScanUtils.empty(repoName)||ScanUtils.empty(mergeId)){
-                    log.error("Namespace/Repo/MergeId must be provided for GITHUBPULL bug tracking");
+                    log.error("--namespace, --repo and --merge-id must be provided for GITHUBPULL bug tracking");
                     exit(1);
                 }
                 mergeNoteUri = gitHubProperties.getMergeNoteUri(namespace, repoName, mergeId);
                 break;
             case GITLABMERGE:
             case gitlabmerge:
```

Comment:
Consider using org.apache.commons.lang3.EnumUtils#getEnumIgnoreCase instead of enum values that only differ in case.